package.json

```diff
@@ -160,7 +161,7 @@
     "gulp-slash": "^1.1.3",
     "lodash.template": "^4.5.0",
     "minimist": "^1.2.5",
-    "native-run": "^1.0.0",
+    "native-run": "1.2.1",
```

Comment:
did this auto bump or did you have to change it?

Reply:
i bumped it. necessary to get to build on an actual device.